tests/unit/models/solid_mechanics/RateIndependentPlasticFlowConstraint.i

```diff
@@ -1,7 +1,7 @@
 [Tensors]
   [fp]
     type = Scalar
-    values = '-2 -1 0 1 2'
+    values = '-2 -1 0.5 1 2'
```

Comment:
Why did you change 0 to 0.5?

Reply:
The FD derivative is off by more than the default tolerance at zero...  This was easier versus increasing the FD tolerance.

Reply:
Okay, I just wanted to make sure it's not a "real" issue.